packages/salesforcedx-vscode/README.md

```diff
@@ -25,6 +25,8 @@ To use Salesforce Extensions for VS Code, install all the extensions in this ext
    This extension uses the Apex Language Server to provide features such as syntax highlighting and code completion.
 * [salesforcedx-vscode-apex-debugger](https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-vscode-apex-debugger)  
    This extension enables VS Code to use the real-time Apex Debugger with your scratch orgs.
+* [salesforcedx-vscode-apex-replay-debugger](https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-vscode-apex-replay-debugger)
```

Comment:
I think this line needs something at the end so that the description shows up on the next line. Looks like I had the same problem at the top level README: https://github.com/forcedotcom/salesforcedx-vscode/blob/ruth/replay-debugger-docs/README.md

![screenshot from 2018-06-06 12 42 30](https://user-images.githubusercontent.com/7286437/41061362-2fb9ae06-6987-11e8-968e-ea16cf69b15a.png)